pkg/module/metrics/dns.go

```diff
@@ -83,7 +84,40 @@ func (d *DNSMetrics) getLabels() []string {
 	return labels
 }
 
-func (d *DNSMetrics) values(flow *v1.Flow) []string {
+func (d *DNSMetrics) getResponseLabels() []string {
+	labels := utils.DNSResponseLabels
+	if d.srcCtx != nil {
```

Comment:
looks like this is also reused in `getRequestLabels`, we can add another helper or modify the func to take in a request/response labels as a paramter